textproc/cjose/Makefile

```diff
@@ -0,0 +1,20 @@
+# $NetBSD$
+
+DISTNAME=       cjose-0.6.1
+CATEGORIES=	textproc
+MASTER_SITES=	${MASTER_SITE_GITHUB:=cisco/}
+
+MAINTAINER=	pkgsrc-users@NetBSD.org
+HOMEPAGE=	https://github.com/cisco/cjose/
+COMMENT=	C implementation of JOSE
+LICENSE=	mit
+
+GNU_CONFIGURE=	yes
+USE_LIBTOOL=	yes
+USE_TOOLS+=	pkg-config
+
+PKGCONFIG_OVERRIDE+=	cjose.pc.in
+
+.include "../../textproc/jansson/buildlink3.mk"
```

Comment:
It's a nit, but long term we prefer to maintain sorted includes (with exception of mk/bsd.pkg.mk that comes the last).

Reply:
Fair enough! Ordered them alphabetically (I assume that's what you mean, right?). 

Maybe this would also be something that could be integrated in to `pkglint`?

Reply:
Right. It would be a valuable addition to `pkglint`. Feel free to report it to the maintainer (or contribute a patch which would be even better).